src/components/App.tsx

```diff
@@ -380,8 +396,15 @@ export class App extends React.Component<any, AppState> {
               collaborators.set(socketID, {});
             }
           }
+          // if alone in the dark, we won't get UDPATE_SCENE init message,
+          //  so initialize early
+          const initializeEarly = !isInitialized && clients.length < 2;
+          if (initializeEarly) {
+            isInitialized = true;
+          }
```

Comment:
If the server is running, we should be guaranteed to get this message, so I put the fallback here. If the server is not running, or there's a problem with the socket, this won't run and the loading state won't go away, but I guess in that case it may be a good thing (which will make user reload or something).

Reply:
This no longer applies because I've added a timer in later commit.